libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -712,42 +712,24 @@ static MQTTStatus_t processPublishFlags( uint8_t publishFlags,
 
 static void logConnackResponse( uint8_t responseCode )
 {
+    const char * const pConnackResponses[ 6 ] =
+    {
+        "Connection accepted.",                               /* 0 */
+        "Connection refused: unacceptable protocol version.", /* 1 */
+        "Connection refused: identifier rejected.",           /* 2 */
+        "Connection refused: server unavailable",             /* 3 */
+        "Connection refused: bad user name or password.",     /* 4 */
+        "Connection refused: not authorized."                 /* 5 */
+    };
```

Comment:
Originally this function was like this, but in a macro guard so that is would only be allocated if logging was enabled. #937 converted this into a switch statement, since that macro is no longer defined anywhere. This reverts it back, since part of that switch was unreachable, and I'm going to trust the compiler to not allocate memory for unused variables.